pkgs/tools/package-management/micromamba/default.nix

```diff
@@ -1,20 +1,12 @@
 { lib, stdenv, fetchFromGitHub, fetchpatch, cmake
-, cli11, nlohmann_json, curl, libarchive, libyamlcpp, libsolv, reproc, spdlog, termcolor, ghc_filesystem
+, cli11, nlohmann_json, curl, libarchive, libyamlcpp, libsolv, reproc, spdlog, termcolor, ghc_filesystem, python3, tl-expected
 }:
 
 let
   libsolv' = libsolv.overrideAttrs (oldAttrs: {
     cmakeFlags = oldAttrs.cmakeFlags ++ [
       "-DENABLE_CONDA=true"
     ];
-
-    patches = [
-      # Patch added by the mamba team
-      (fetchpatch {
-        url = "https://raw.githubusercontent.com/mamba-org/boa-forge/20530f80e2e15012078d058803b6e2c75ed54224/libsolv/conda_variant_priorization.patch";
-        sha256 = "1iic0yx7h8s662hi2jqx68w5kpyrab4fr017vxd4wyxb6wyk35dd";
-      })
-    ];
```

Comment:
Has this patch been removed/deprecated? Does it break the build with the latest changes?

Reply:
The patch is still there in master : https://github.com/mamba-org/boa-forge/blob/master/libsolv/conda_variant_priorization.patch
Commit message doesn't say much https://github.com/mamba-org/boa-forge/commit/20530f80e2e15012078d058803b6e2c75ed54224 :roll_eyes: 

If it doesn't break the build let's keep this patch, if only just to keep parity with the official binaries of micromamba.